static/app/components/events/interfaces/frame/deprecatedLine.tsx

```diff
@@ -305,7 +305,11 @@ export class DeprecatedLine extends Component<Props, State> {
               {t('Suspect Frame')}
             </SuspectFrameTag>
           ) : null}
-          {!data.inApp ? <Tag>{t('System')}</Tag> : <Tag type="info">{t('In App')}</Tag>}
+          {!data.inApp ? (
+            <InAppTag>{t('System')}</InAppTag>
```

Comment:
`InAppTag` is a bit of a misnomer here since it can have system tags too, but I'll allow it since we are going to remove those soon 😄